osgi/felixtest/pom.xml

```diff
@@ -111,6 +111,7 @@
                         <id>felix-integration-test</id>
                         <goals>
                             <goal>integration-test</goal>
+                            <goal>verify</goal>
```

Comment:
Uff.

Reply:
Yeah :). To be fair, it's not the most natural setup.